flexible_event_config.md

```diff
@@ -101,6 +102,21 @@ In addition to the parameters that were added in Phase 1, we will add one additi
     // Next trigger_spec
   }, ...],
 
+  // Specifies how the 64-bit unsigned trigger_data from the trigger is matched
+  // against the source's trigger_specs trigger_data, which is 32-bit. Defaults
+  // to "modulus".
+  //
+  // If "exact", the trigger_data must exactly match a value contained in the
+  // source's trigger_specs; if there is no such match, no event-level
+  // attribution takes place.
+  //
+  // If "modulus", the source's trigger specs must contain trigger_data forming
```

Comment:
This is slightly unclear to me. The example below shoes that the set of all trigger_data values across all trigger_specs must be a contiguous sequence starting at 0. Could we rephrase to make this more clear?